### DISCUSSION TITLE
Update riverpod_lint ?

### DISCUSSION THREAD

**Discussion by CharlotteJu** [rrousselGit/riverpod#2775] [Q&A]
Hello,

Do you know when a new version compatible with riverpod_lint 1.3.12 or 1.4.0 is planned? 
I'm experiencing dependency conflicts with the analyzer package and several other libraries because of this...

Best regards,

**Comment by rrousselGit** [comment]
analyzer_plugin needs to be updated to support analyzer 6.0.0

Nothing I can do in the meantime. It's the Dart team that's blocking

**Comment by CharlotteJu** [comment]
Sure, thank you. I hope you'll manage to do it :)
